valkey.conf

```diff
@@ -329,7 +329,7 @@ daemonize no
 #
 # When the server runs non daemonized, no pid file is created if none is
 # specified in the configuration. When the server is daemonized, the pid file
-# is used even if not specified, defaulting to "/var/run/redis.pid".
+# is used even if not specified, defaulting to "/var/run/valkey.pid".
```

Comment:
```suggestion
# is used even if not specified, defaulting to "/var/run/redis.pid".
```

Reply:
Thanks for catching this one too @madolson. Apparently I am OOM'ing on all the compat decisions ...